EnglishFilter/sections/specific.txt

```diff
@@ -243,6 +243,8 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
 malaysiastock.biz###sideBar > div[class^="RightPanel_Rectangle"]
 ehow.com##.ad
 ||download.oxy.st/dd.png
+speed-down.org##div[style*="width:336px;"][style*="height:280px;"]
```

Comment:
it is hidden on my end

Reply:
https://speed-down.org/download2353052
check this link

Reply:
the same 
![image](https://user-images.githubusercontent.com/8361299/88304323-c9617680-cd10-11ea-8b22-73635d7e51d5.png)

Reply:
This rule was added by Blaz today and there is some problems with the site, download links don't work

Reply:
Original link not available anymore and is redirected to ad (fake one). That one does not produce correct download link without AdGuard.